src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,21 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    public String[] readFromFile(String fileName) throws RuntimeException {
+        try {
+            List<String> sortedWordsWithW = new ArrayList<>();
+            for (String line : Files.readAllLines(Paths.get(fileName))) {
+                for (String word : line.toLowerCase().split(" ")) {
+                    if (word.substring(0, 1).matches("w")) {
+                        sortedWordsWithW.add(word.replaceAll("\\W+",""));
+                    }
+                }
+            }
+            Collections.sort(sortedWordsWithW);
+            String[] result = sortedWordsWithW.toArray(new String[0]);
+            return result;
+        } catch (IOException e) {
+            throw new RuntimeException();
```

Comment:
any exception should be informative, right?
```suggestion
            throw new RuntimeException(e);
```

Reply:
@fyarkiy fix this comment too